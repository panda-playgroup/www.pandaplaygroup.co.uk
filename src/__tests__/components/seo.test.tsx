import { render } from "@testing-library/react";
import Seo from "../../components/seo";

jest.mock("../../hooks/use-site-metadata", () => ({
    useSiteMetadata: jest.fn().mockReturnValue({ title: "Test Title", description: "Test Description" }),
}));

describe("seo", () => {
    it("renders", () => {
        const { asFragment } = render(<Seo />);

        expect(asFragment()).toMatchSnapshot();
    });

    it("renders with a page title", () => {
        const { asFragment } = render(<Seo pageTitle="Test Page Title" />);

        expect(asFragment()).toMatchSnapshot();
    });
});
