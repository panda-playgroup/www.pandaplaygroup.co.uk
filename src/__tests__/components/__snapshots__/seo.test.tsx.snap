// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`seo renders 1`] = `
<DocumentFragment>
  <title>
    Test Title
  </title>
  <meta
    content="Test Description"
    name="description"
  />
</DocumentFragment>
`;

exports[`seo renders with a page title 1`] = `
<DocumentFragment>
  <title>
    Test Page Title | Test Title
  </title>
  <meta
    content="Test Description"
    name="description"
  />
</DocumentFragment>
`;
